server/test/conf/MessagesTest.java

```diff
@@ -37,28 +37,8 @@ public class MessagesTest {
   // we are waiting for translations for.
   private static final ImmutableSet<String> IGNORE_LIST =
       ImmutableSet.of(
-          // TODO(#4592): Remove once these are translated
-          "button.continueWithoutAnAccount",
-          "content.loginModalPrompt",
-          "button.applyToPrograms",
-          "button.createAnAccount",
-          "content.commonIntakeConfirmation",
-          "content.commonIntakeConfirmationTi",
-          "content.commonIntakeNoMatchingPrograms",
-          "content.commonIntakeNoMatchingProgramsNextStep",
-          "content.commonIntakeNoMatchingProgramsTi",
-          "content.guestDescription",
-          "content.saveTime",
-          "title.commonIntakeConfirmation",
-          "title.commonIntakeConfirmationTi",
-          "content.adminFooterPrompt",
-          "label.countryCode",
-          "label.phoneNumber",
-          "validation.invalidPhoneNumberProvided",
-          "validation.phoneCountryCodeRequired",
-          "validation.phoneMustBeLocalToCountry",
-          "validation.phoneNumberMustContainNumbersOnly",
-          "validation.phoneNumberRequired");
+          // TODO(#4640): Translations Batch 6 - Remove once these are translated
```

Comment:
Remove this comment? I think it's confusing to leave it here when there aren't actually any ignored keys.

Reply:
Updated to be an instruction to insert them below.

I want to leave it since we might often have keys in this list from different translation batches at the same time, and it would help to keep them separated.